samples/bluetooth/enocean/README.rst

```diff
@@ -0,0 +1,114 @@
+.. _enocean_sample:
+
+Bluetooth: EnOcean
+##################
+
+The Bluetooth EnOcean sample demonstrates the basic usage of the :ref:`bt_enocean_readme` library.
+
+Overview
+********
+
+The EnOcean sample sets up a basic Bluetooth observer for both EnOcean switches and sensors.
+
+.. _enocean_sample_overview:
+
+.. include:: ../../../include/bluetooth/enocean.rst
+   :start-after: enocean_observing_output_start
+   :end-before: enocean_observing_output_end
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF5340DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+* At least one :ref:`supported EnOcean device <bt_enocean_devices>`.
+
+.. note::
+    The sample supports up to four devices at a time that work with one board.
+    Using devices to work with different boards (or more than one board) is not supported.
+
+User interface
+**************
+
+The following board LEDs are used by this sample:
+
+LED 1 and LED 2:
+   At the end of commissioning, blink four times to indicate a new light switch device has been commissioned.
+   
+   After commissioning, indicate whether a button is pressed on a commissioned light switch device.
+   
+   .. note::
+        If you are using EnOcean switches with a single rocker, toggling the rocker is indicated on LEDs 2 and 4.
+
+LED 3 and LED 4:
+   Indicate the On/Off state of each button channel.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/enocean`
+
+.. include:: /includes/build_and_run.txt
+
+.. _enocean_sample_testing:
+
+Testing
+=======
+
+After programming the sample to your board, test it by performing the following steps:
+
+1. :ref:`Commission one or more EnOcean devices <bt_enocean_commissioning>`.
+   The board LEDs will blink when each of the devices has been successfully commissioned.
+#. Connect the board to the computer with an USB cable.
```

Comment:
Nit: This step is optional for light switches, the logs don't really add that much.